modules/private-shared-lib/src/domain/bulk_transaction_agg/index.ts

```diff
@@ -167,23 +165,4 @@ export class BulkTransactionAgg extends BaseAggregate<BulkTransactionEntity, Bul
             // this._partyLookupFailedCount = undefined;
         }
     }
-
-
-    static async ProcessCommandEvent(
```

Comment:
Why was this removed?

Reply:
It has been moved here https://github.com/mojaloop/sdk-scheme-adapter/pull/354/files#diff-7d1faa4862c22f5fcf174cadd8101d3cf5a0ede1e347c600ec78bd2758340bb5R92 I saw no reason it was a static method on this class so I moved it to the handler.

Reply:
`bulk_transaction_agg` <-- SHOULD only exist in the Command handler.

I believe you were ONLY suppose to move the REPO, NOT Aggregates!